DataMigration/DataLayer/PostgresConnect.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Npgsql;
+using DataMigration.PostgresDB;
+using DataMigration.Logger;
+
+
+namespace DataMigration.DataLayer
+{
+    public class PostgresConnect
+    {
+        private Logger.Logger _log = new Logger.Logger();
+
+        public void StartLogConsole(Logger.Logger logger)
+        {
+            _log = new Logger.Logger(logger);
+        }
+
+        public int GetCountOfHistoricData(string connectString)
+        {
+            var res=0;
+            using (var conn = new NpgsqlConnection(connectString))
+            {
+                conn.Open();
+                using (var command = GetCommand(conn, "select count(*) from historical_ocr_data"))
+                {
+                    using (var reader = command.ExecuteReader())
+                    {
+                        while (reader.Read())
+                        {
+                            res = reader.GetInt32(0);
+                        }
+                    }
+                }
+            }
+            return res;
+        }
+        public List<string> GetAllHistoricPaths(string connectString,int limit)
+        {
+            var docs=new List<string>();
+            try
+            {
+                _log.WriteLog(LogLevel.Info, $"Get {limit} historic paths (Postgres)\n");
+                using (var conn = new NpgsqlConnection(connectString))
+                {
+                    conn.Open();
+                    using (var command = GetCommand(conn, $"SELECT fullfilepath FROM historical_ocr_data LIMIT {limit}"))
+                    {
+                        using (var reader = command.ExecuteReader())
+                        {
+                            while (reader.Read())
+                            {
+                                docs.Add(ReplaceUnsupportedCharacters(reader.GetString(0), "/", "\\"));
+                            }
+                        }
+                    }
+                }               
+            }
+            catch (Exception ex)
+            {
+                _log.WriteLog(LogLevel.Info, "Something wrong with executing command or with data in table \n");
+                _log.WriteLog(LogLevel.Error, ex.Message + "\n" + ex.StackTrace);
```

Comment:
error log is ok (info log is redundant)
"Something wrong with executing command or with data in table \n" is not informative enough. 
should be something like "Error while getting all historical paths. Error details:  "+ ex.Message.